proxy/pkg/metrics/node_metrics.go

```diff
@@ -280,7 +280,7 @@ type NodeMetricsInstance struct {
 	UnavailableErrors Counter
 	OtherErrors       Counter
 
-	RequestDuration Histogram
+	RequestDuration map[string]Histogram
```

Comment:
Why not follow what we have on `ProxyMetrics`? We already have write and read metrics on the `ProxyMetrics` , just need to replicate what's there.